client/src/endpoint/payment.rs

```diff
@@ -235,6 +235,24 @@ impl IntoRequest for FindPath {
     }
 }
 
+impl TryFromUri for FindPath {
+    fn try_from_wrap(wrap: &UriWrap) -> ::std::result::Result<FindPath, uri::Error> {
+        let params = wrap.params();
+        let destination_asset = AssetIdentifier::new(
+            params.get_ok("destination_asset_type")?,
+            params.get_parse("destination_asset_code").ok(),
```

Comment:
These are just strings right? Do you need to get parse them?

Reply:
Yes, true. Maybe I am missing something, but when I do:

```rust
params.get("destination_asset_code")
```

I get the following error:
```
note: expected type `std::option::Option<std::string::String>`
               found type `std::option::Option<&str>`
```

Reply:
Ah, it wants it to be owned, so you are actually parsing it. Carry on, nothing to see here.